### DISCUSSION TITLE
special cause concern/improvement

### DISCUSSION THREAD

**Discussion by send2dan** [nhs-r-community/NHSRplotthedots#161] [General]
Hello,

There are several different rulebases available/published that define special cause variation. 

With regards to this package, I'm trying to find the answer to the following statements:
1) Special cause improvement is defined as by the {NHSRplotthedots} package as... 
2) Special cause concern is defined as by the {NHSRplotthedots} package as... 

I can't seem to (easily) find this information in the R help files for this package, but I might be looking in the wrong place?

Best wishes,

Dan

**Comment by ThomUK** [comment]
Hi Dan,
Thanks for using the package, and for the feedback.  Yes, this probably needs including in the documentation.

Points are special cause if any of the following criteria are satisfied:

* Any point beyond the dotted control limits.

* 2 out of the last 3 points are beyond 2sigma (ie. 2/3 of the way from mean to control limit).  Only the 2 points meeting the criteria are highlighted.

* 7 or more points on one side of the mean.  In this case all 7 qualifying points are highlighted (or more if it's more points).

Whether they are blue (improvement), orange (concern), or purple (neutral) depends on which way the "improvement_direction" argument is set in the ptd_spc() function call. Up good, down bad, etc.
